src/feature/dirauth/process_descs.c

```diff
@@ -307,9 +306,11 @@ dirserv_router_get_status(const routerinfo_t *router, const char **msg,
   }
 
   /* Check for the more usual versions to reject a router first. */
```

Comment:
This comment doesn't make sense.
Let's fix it while we are here?

"First, check for the more common reasons to reject a router"